cmd/crictl/pod_metrics.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+	"golang.org/x/net/context"
+	cri "k8s.io/cri-api/pkg/apis"
+	pb "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+type podMetricsOptions struct {
+	// output format
+	output string
+
+	// live watch
+	watch bool
+}
+
+var podMetricsCommand = &cli.Command{
+	Name:                   "metricsp",
+	Aliases:                []string{"metrics"},
+	Usage:                  "List pod resource usage statistics",
```

Comment:
the description is exactly like `statsp` https://github.com/kubernetes-sigs/cri-tools/blob/94d44a2463ce016540effcc4cf219a2e6630753e/cmd/crictl/pod_stats.go#L52

Should there be any differenece?

Reply:
good call, I updated to `List pod resource usage metrics` which isn't super informative but I think will differentiate enough

Reply:
we really need to expand on both descriptions. I cannot imagine anybody figuring the difference on their own.